src/index.html

```diff
@@ -5,8 +5,439 @@
+              <div class="stars__star"></div>
+              <div class="stars__star"></div>
+              <div class="stars__star"></div>
+              <div class="stars__star"></div>
+            </div>
+    
+            <div class="product-card__responce">
+              Reviews: 5
+            </div>
+          </div>
+    
+          <div class="product-card__price">
+            <div class="product-card__title">
+              Price:
+            </div>
+    
+            <div class="product-card__value">
+              $2,199
+            </div>
+          </div>
+    
+          <a 
+            class="product-card__button"
+            href="#"
+          >
+          buy
+          </a>
+        </div>
+
+
+        <div class="product-card">
+          <img
+            class="product-card__img" 
+            src="images/imac.jpeg"
+            alt="imac"
+          >
+    
+          <div class="product-card__name">
+            APPLE A1419 iMac 27" Retina <br> 5K Monoblock (MNED2UA/A)
+          </div>
+    
+          <div class="product-card__code">
+            Product code: 195434
+          </div>
+    
+          <div class="product-card__stars">
+            <div class="stars stars--4">
+              <div class="stars__star"></div>
+              <div class="stars__star"></div>
+              <div class="stars__star"></div>
+              <div class="stars__star"></div>
+              <div class="stars__star"></div>
+            </div>
+    
+            <div class="product-card__responce">
+              Reviews: 5
+            </div>
+          </div>
+    
+          <div class="product-card__price">
+            <div class="product-card__title">
+              Price:
+            </div>
+    
+            <div class="product-card__value">
+              $2,199
+            </div>
+          </div>
+    
+          <a 
+            class="product-card__button"
+            href="#"
+          >
+          buy
+          </a>
+        </div>
+
+        <div class="product-card">
+          <img
+            class="product-card__img" 
+            src="images/imac.jpeg"
+            alt="imac"
+          >
+    
+          <div class="product-card__name">
+            APPLE A1419 iMac 27" Retina <br> 5K Monoblock (MNED2UA/A)
+          </div>
+    
+          <div class="product-card__code">
+            Product code: 195434
+          </div>
+    
+          <div class="product-card__stars">
+            <div class="stars stars--4">
+              <div class="stars__star"></div>
+              <div class="stars__star"></div>
+              <div class="stars__star"></div>
+              <div class="stars__star"></div>
+              <div class="stars__star"></div>
+            </div>
+    
+            <div class="product-card__responce">
+              Reviews: 5
+            </div>
+          </div>
+    
+          <div class="product-card__price">
+            <div class="product-card__title">
```

Comment:
```suggestion
            <div class="product-card__price-title">
```
`product-card__title` - seems that is title of the whole card, not only the price